ClientApp/src/app/models/person.class.ts

```diff
@@ -0,0 +1,136 @@
+interface PersonBase {
+  getDisplayName(property: string): string;
+}
```

Comment:
I don't like `getDisplayName`, because that sounds almost like internationalization (but not properly), but imo thats something we should implement later

Reply:
okay, I am not a fan of it either of it.
I think I will hardcode these in the view.